zproject/prod_settings_template.py

```diff
@@ -449,6 +449,17 @@
     ## set this to True to enable signing of SAMLRequests using the
     ## private key.
     "authnRequestsSigned": False,
+    ## You can change requestedAuthnContext to True to make Zulip include AuthnContext in
+    ## SAMLRequests, specifying AuthnContextClassRef. By default,
+    ## it will only permit urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport.
+    ## You can also set this to a list of specific Authentication Context Classes that you want
+    ## to include in the AuthnContext. E.g.:
+    ## "requestedAuthnContext": ["urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport",
+    ##                           "urn:oasis:names:tc:SAML:2.0:ac:classes:X509"]
+    ## Refer to python3-saml documentation for this setting
```

Comment:
```suggestion
    ## Refer to python-saml documentation for this setting
```

And maybe link to https://github.com/onelogin/python-saml/blob/master/README.md#settings above this whole block?

Reply:
Yeah, I think adding a link to that is a good idea.